h/static/styles/core/_typography.scss

```diff
@@ -115,3 +157,17 @@
     border-radius: 2px;
   }
 }
+
+// Rules
+// ----------------------------------------------------------------------------
+
+// Explanatory text paragraphs
+.text {
+  color: $grey-6;
+}
+
+// Prompt used in the OAuth "Authorize access?" screen.
+.text--auth-prompt {
+  color: $grey-5;
+  margin-bottom: 50px;
+}
```

Comment:
This rule is specific to a particular use case. It probably ought to be its own component.

Reply:
This needs to be deleted completely*, but for the purposes of this PR I only deleted CSS that wasn't invoked anywhere. I put it in the `_typography` file because of the classname, even though it also contains a layout declaration (`margin-bottom`).

* `.text` and `.text--auth-prompt` are both only used once, and they're on the same element; `text` is too vague for a block name, etc.